tools/codegen/cmd/featuregate-test-analyzer.go

```diff
@@ -543,7 +543,17 @@ func getLatestRelease() (string, error) {
 		return "", fmt.Errorf("no releases found")
 	}
 
-	return result.Releases[0], nil
+	// if its not main branch, then use the ENV var to determine the release version
+	currentRelease := os.Getenv("PULL_BASE_REF")
+	if len(currentRelease) == 0 {
+		return "", fmt.Errorf("unable to fetch the release version")
+	}
```

Comment:
If we move this first, we can re-organise this and just fall back to sippy when the env var isn't set, I think that's a safer assumption to make

Reply:
done.